### ISSUE TITLE
cannot create new group + other issues adding group to user in NC14.0.0

### ISSUE THREAD

**Issue by rollanders** [nextcloud/server#11182]
### Steps to reproduce
1. Open Users page
2. Add New user
3. Go to "Add user in group". There are two alternatives - admin and Everyone. 
4. Only admin can be selected. If I by mistake select that and want to deselect I cannot
5. Write the name for a new group in "Add user in group" 
6. I get a response "No results"
7. I do not find any means to create a new group in the UI
8. I can set the new user as administrator for the Everyone group, but cannot add him to the Everyone group

### Expected behaviour
I should be able to deselect the admin group from the new user. 
I should be able to and create a new group for the new user
I should be able to add the new user to the Everyone group
 

### Actual behaviour
I cannot deselect the admin group from the new user
I cannot create a new group
I cannot add the new user to the Everyone group

## Server configuration detail

**Operating system:** Linux 4.15.0-23-generic #25-Ubuntu SMP Wed May 23 18:02:16 UTC 2018 x86_64

**Webserver:** nginx/1.15.1 (fpm-fcgi)

**Database:** mysql 10.1.34

**PHP version:** 

7.2.7-1+ubuntu18.04.1+deb.sury.org+1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, cgi-fcgi, mysqlnd, PDO, xml, apcu, apc, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, igbinary, imagick, intl, json, ldap, exif, mysqli, pdo_mysql, Phar, posix, readline, redis, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Zend OPcache

**Nextcloud version:** 14.0.0 - 14.0.0.19

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.0.1
 - activity: 2.7.0
 - admin_audit: 1.4.0
 - audioplayer: 2.3.2
 - calendar: 1.6.2
 - caniupdate: 0.2.0
 - cloud_federation_api: 0.0.1
 - cms_pico: 0.9.7
 - comments: 1.4.0
 - contacts: 2.1.6
 - dav: 1.6.0
 - external: 3.1.0
 - federatedfilesharing: 1.4.0
 - federation: 1.4.0
 - files: 1.9.0
 - files_pdfviewer: 1.3.2
 - files_rightclick: 0.8.4
 - files_sharing: 1.6.2
 - files_texteditor: 2.6.0
 - files_trashbin: 1.4.1
 - files_versions: 1.7.1
 - files_videoplayer: 1.3.0
 - gallery: 18.1.0
 - group_everyone: 0.1.0
 - groupfolders: 1.3.3
 - issuetemplate: 0.4.0
 - logreader: 2.0.0
 - lookup_server_connector: 1.2.0
 - mail: 0.10.0
 - nextcloud_announcements: 1.3.0
 - notes: 2.4.1
 - notifications: 2.2.1
 - oauth2: 1.2.1
 - onlyoffice: 2.0.4
 - ownpad: 0.6.8
 - password_policy: 1.4.0
 - provisioning_api: 1.4.0
 - richdocuments: 2.0.12
 - serverinfo: 1.4.0
 - sharebymail: 1.4.0
 - support: 1.0.0
 - systemtags: 1.4.0
 - theming: 1.5.0
 - twofactor_backupcodes: 1.3.1
 - updatenotification: 1.4.1
 - workflowengine: 1.4.0
Disabled:
 - encryption
 - files_external
 - firstrunwizard
 - impersonate
 - survey_client
 - user_external
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "XXXXXX.dedyn.io"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/XXXXXX.dedyn.io",
    "dbtype": "mysql",
    "version": "14.0.0.19",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "rewrite_url": true,
    "activity_expire_days": 14,
    "auth.bruteforce.protection.enabled": true,
    "blacklisted_files": [
        ".htaccess",
        "Thumbs.db",
        "thumbs.db"
    ],
    "cron_log": true,
    "enable_previews": true,
    "enabledPreviewProviders": [
        "OC\\Preview\\PNG",
        "OC\\Preview\\JPEG",
        "OC\\Preview\\GIF",
        "OC\\Preview\\BMP",
        "OC\\Preview\\XBitmap",
        "OC\\Preview\\Movie",
        "OC\\Preview\\PDF",
        "OC\\Preview\\MP3",
        "OC\\Preview\\TXT",
        "OC\\Preview\\MarkDown"
    ],
    "filesystem_check_changes": 0,
    "filelocking.enabled": "true",
    "htaccess.RewriteBase": "\/",
    "integrity.check.disabled": false,
    "knowledgebaseenabled": false,
    "logtimezone": "Europe\/Berlin",
    "log_rotate_size": 104857600,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "preview_max_x": 1024,
    "preview_max_y": 768,
    "preview_max_scale_factor": 1,
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0,
        "timeout": 0
    },
    "quota_include_external_storage": false,
    "share_folder": "\/Shares",
    "skeletondirectory": "",
    "trashbin_retention_obligation": "auto, 7",
    "maintenance": false,
    "theme": "",
    "loglevel": 2,
    "updater.release.channel": "stable"
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36

**Operating system:** Windows 10 - updated

## Logs

<details><summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

	?????
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here

Error	PHP	Undefined index: log-detail at /var/www/nextcloud/apps/issuetemplate/lib/Detail.php#59
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	?????
</details>

**Comment by nickvergessen** [comment]
@skjnldsv mind to tackle it?

**Comment by skjnldsv** [comment]
Ah yes, indeed. I'm confused. :thinking:

**Comment by acs-ferreira** [comment]
Confirm this, no **Groups** option is available anywhere (create, manage...) in NC14.

**Comment by skjnldsv** [comment]
> Only admin can be selected. If I by mistake select that and want to deselect I cannot

After you clicked admin, there is a small tick icon stating that the group is selected. If you click it again, it will be unselected.

> I cannot create a new group

You can by entering the name of your group in the dropdown and press enter.

> I cannot add the new user to the Everyone group

This is not a group. You cannot add a user to the list of all users, he will be part of it anyway.

**Comment by acs-ferreira** [comment]
> You can by entering the name of your group in the dropdown and press enter.

Hum. Ok. But how to create groups independently of user creation?
Possible to have a button just like NC13?

**Comment by skjnldsv** [comment]
@acs-ferreira this is the only missing feature.
Will be in 14.0.1

**Comment by rollanders** [comment]
It works. I first need to create a user without specifying a group. After having created the new user he is only active in the Everyone group. 
After the user has been created I can create a new group for the new user in the field that represent "Add user in group". Not intuitive however.

**Comment by somoza** [comment]
> 
> > I cannot create a new group
> 
> You can by entering the name of your group in the dropdown and press enter.
> 

That is nothing intuitive...

**Comment by rollanders** [comment]
![image](https://user-images.githubusercontent.com/36486009/45900270-916d0d80-bddf-11e8-8dee-c032445db613.png)

This is how it looks if I do just that. I add name of the new user, display name, and try to add new grou. I get "No result". As I wrote I foirst need to create the new group with admin as user. Then I can addd a new user and assign this new group.

**Comment by obendev** [comment]
So is this fixed in 14.0.1 now or do we have to wait for 14.0.2?
